src/app/header/header.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, OnInit } from '@angular/core';
+import {Router} from '@angular/router';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss']
+})
+export class HeaderComponent implements OnInit {
+  public breadcrumb: string;
+
+  constructor(private router: Router) {
+    this.router.events.subscribe(_ => {
+      this.breadcrumb = this.router.routerState.snapshot.url;
+      const length = this.breadcrumb.split('/').length;
+      document
+        .getElementById('breadcrumb')
```

Comment:
Avoid this. Put the logic inside a function in this component, and use the result in ng-for in .html